package.json

```diff
@@ -11,9 +11,11 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "ajv": "^8.11.0",
+    "aws-sdk": "^2.1362.0",
```

Comment:
Since the data is being pulled from a public bucket, there isn't a need for aws-sdk